src/game/server/gamemodes/insta_core/insta_core.cpp

```diff
@@ -1187,18 +1188,21 @@ void CGameControllerInstaCore::ApplyVanillaDamage(int &Dmg, int From, int Weapon
 		GameServer()->CreateSound(pCharacter->m_Pos, SOUND_PLAYER_PAIN_SHORT);
 }
 
-void CGameControllerInstaCore::MakeLaserTextPoints(vec2 Pos, int Points, int Seconds, CClientMask Mask)
+void CGameControllerInstaCore::MakeTextPoints(vec2 Pos, int Owner, int Points, int Seconds, int TextType) const
 {
-	if(!g_Config.m_SvLaserTextPoints)
+	if(!g_Config.m_SvTextPoints)
 		return;
 
 	char aText[16];
 	if(Points >= 0)
 		str_format(aText, sizeof(aText), "+%d", Points);
 	else
-		str_format(aText, sizeof(aText), "%d", Points);
+		str_format(aText, sizeof(aText), "-%d", Points);
```

Comment:
Isn't that two `-` now? The `%d` of `str_format` should already display one `-` if this is false `Points >= 0`